CREDITS

```diff
@@ -84,12 +84,18 @@ research project called The Harvest Information Discovery and Access System:
 
 ==============================================================================
 
-acinclude/ax_cxx_compile_stdcxx_11.m4:
+acinclude/ax_cxx_compile_stdcxx.m4:
 
 #   Copyright (c) 2008 Benjamin Kosnik <bkoz@redhat.com>
 #   Copyright (c) 2012 Zack Weinberg <zackw@panix.com>
 #   Copyright (c) 2013 Roy Stogner <roystgnr@ices.utexas.edu>
-#   Copyright (c) 2014 Alexey Sokolov <sokolov@google.com>
+#   Copyright (c) 2014, 2015 Google Inc.; contributed by Alexey Sokolov <sokolov@google.com>
+#   Copyright (c) 2015 Paul Norman <penorman@mac.com>
+#   Copyright (c) 2015 Moritz Klammler <moritz@klammler.eu>
+#   Copyright (c) 2016, 2018 Krzesimir Nowak <qdlacz@gmail.com>
+#   Copyright (c) 2019 Enji Cooper <yaneurabeya@gmail.com>
+#   Copyright (c) 2020 Jason Merrill <jason@redhat.com>
+#   Copyright (c) 2021 Jörn Heusipp <osmanx@problemloesungsmaschine.de>
```

Comment:
This entry copies a non-ASCII character from the corresponding source file, but our CREDITS file has two other entries with non-ASCII characters, so I think that copying is the right thing to do (at least for this file/PR).